Projects/Foundations/Schemas/Docs-Books/first-steps-tutorials.json

```diff
@@ -208,11 +248,6 @@
             "text": "Foundations->Concept->List of Plugin Workspaces and Tutorials->Content",
             "updated": 1610955413534
         },
-        {
```

Comment:
Was it intended byyou to remove that part?
@justuser668 ustuser668

Reply:
was a fail, is changed in newer translations

Reply:
For me it doesnt seem like this part changed,did you app.update after fixing the problem?

Reply:
jetzt aber, entschuldige war gestern ein unbedachter Mausklick...